src/Symfony/Component/Console/Helper/Table.php

```diff
@@ -48,7 +51,14 @@ class Table
      * Table rows.
      */
     private $rows = [];
-    private $horizontal = false;
+
+    /**
+     * Defines the way the table will be displayed.
+     * It can be either 'default', 'horizontal' or 'vertical'.
+     *
+     * @var string
+     */
+    private $displayOrientation = self::DISPLAY_ORIENTATION_DEFAULT;
```

Comment:
You can add type to properties.
```suggestion
    private string $displayOrientation = self::DISPLAY_ORIENTATION_DEFAULT;
```

Reply:
Done